components/templates/inverstor/investor.tsx

```diff
@@ -1,155 +1,228 @@
-import Subtitle2 from '../../text/subtitle2'
+import Subtitle from '../../text/subtitle'
 import Title from '../../text/title'
-import Benefit from './benefit'
-import NFT from './nft'
-import { IBenefit, INFT } from './types'
-
-const nfts: INFT[] = [
-  {
-    name: 'Fish',
-    category: 'Silver',
-    crypto: IconCryptoBtc,
-    progress: 'up',
-    imageUrl: '/nfts/fish/silver/happy.png',
-  },
-  {
-    name: 'Dragon',
-    category: 'Platinium',
-    crypto: IconCryptoEth,
-    progress: 'stable',
-    imageUrl: '/nfts/dragon/platinum/normal.png',
-  },
-  {
-    name: 'Bull',
-    category: 'Bronze',
-    crypto: IconCryptoBtc,
-    progress: 'down',
-    imageUrl: '/nfts/bull/bronze/angry.png',
-  },
-  {
-    name: 'Deer',
-    category: 'Gold',
-    crypto: IconCryptoEth,
-    progress: 'down',
-    imageUrl: '/nfts/deer/gold/angry.png',
-  },
-  {
-    name: 'Bear',
-    category: 'Diamond',
-    crypto: IconCryptoBtc,
-    progress: 'stable',
-    imageUrl: '/nfts/bear/diamond/normal.png',
-  },
-  {
-    name: 'Whale',
-    category: 'Diamond',
-    crypto: IconCryptoEth,
-    progress: 'up',
-    imageUrl: '/nfts/whale/diamond/happy.png',
-  },
-]
-
-const benefits: IBenefit[] = [
-  {
-    name: 'Emotional NFT',
-    category: 'Community Art NFTs',
-    items: [
-      'Lovely art for stressless investment',
-      'Get access to locked assets sale',
-      'Community gamification',
-    ],
-  },
-  {
-    name: 'QREP Token',
-    category: 'Reputation Token',
-    items: [
-      'Get initial reputation in the ecosystem with QREP Token',
-      'Access DICP voting power',
-      'Unlock DICP protocol rewards',
-      'Trust from the community',
-    ],
-  },
-  {
-    name: 'QSTK Token',
-    category: 'Economy Token',
-    items: [
-      'Receive QSTK Token at a discounted price (locked asset)',
-      'Access Quiver IDAO voting power',
-      'Get discounted price on information purchase from the DICP',
-      'Unlock DICP protocol rewards',
-    ],
-  },
-]
+import Steps from './steps'
+
+const Info = (
+  <>
+    <h5 className="text-sm leading-5 font-medium text-purple-900">
+      Get a better deal with Emotional NFTs
+    </h5>
+    <p className="mt-2 text-xs leading-4 font-normal text-purple-900">
+      Did you know you can get a higher discount on your QSTK Token purchase?
+    </p>
+    <a
+      href="https://emotional.quiverprotocol.com"
+      className="mt-4 text-xs leading-4 font-medium text-purple-700"
+    >
+      Check out our Emotional NFTs
+      <ArrowNarrowRightIcon className="ml-2 w-3 h-3 inline-flex" />
+    </a>
+  </>
+)
 
 const Investor: FunctionComponent = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false)
+  const [amount, setAmount] = useState<BigNumber>()
+  const [tx, setTx] = useState<ContractTransaction>()
+  const [receipt, setReceipt] = useState<ContractReceipt>()
+
+  const { activate, chainId: sellChainId } = useWeb3React<Web3Provider>()
```

Comment:
`sellChainId` is not a great name and it should be spelled `saleChainId`.
Maybe focusing on the user vs remote provider? eg:
```suggestion
  const { activate, chainId: remoteChainId } = useWeb3React<Web3Provider>()
```
and the one from the user should also be renamed to `userChainId` to be super explicit